src/generated/drupal.ts

```diff
@@ -1,14 +1,14 @@
 export const DRUPAL_THEME_FILTER_IDS = new Map([
```

Comment:
Why have these ids suddenly changed?

Reply:
I've absolutely no idea, I reverted this now. Going to look into it to fully understand how this is generated